src/application-base/private/getBrowserLocale.js

```diff
@@ -24,6 +24,13 @@ const filterLanguages = () => {
   if (navigator.languages && navigator.languages.length > 0) {
     for (let index = 0; index < navigator.languages.length; index += 1) {
       const locale = navigator.languages[index];
+      const fallbackLocale = locale.split('-').length > 1 ? locale.split('-')[0] : false;
+
+      if(fallbackLocale){
+        if (isSupported(fallbackLocale)) {
+          return fallbackLocale;
+        }
+      }
 
       if (isSupported(locale)) {
```

Comment:
We need to check the preferred locale prior to falling back to the regional locale

Reply:
changed here 7731a62c36827a92ca385599cec577dc79d6fd1e